modules/schedule_module/jsx/scheduleIndex.js

```diff
@@ -0,0 +1,565 @@
+          <TimeElement
+            name = "AppointmentTime"
+            label = "Appointment Time"
+            onUserInput = {this.setFormData}
+            value = {this.state.formData.AppointmentTime}
+            required = {false}
+          />
+          <SelectElement
+            name="AppointmentType"
+            label = "Appointment Type"
+            options={this.state.data.fieldOptions.AppointmentType}
+            value={this.state.formData.AppointmentType}
+            required={true}
+            onUserInput={this.setFormData}
+          />
+        {this.renderScheduleFormButton()}
+        </FormElement>
+      </Modal>
+    );
+  }
+  render() {
+    // If error occurs, return a message.
+    // XXX: Replace this with a UI component for 500 errors.
+    if (this.state.error) {
+      return <h3>An error occured while loading the page.</h3>;
+    }
+
+    // Waiting for async data to load
+    if (!this.state.isLoaded) {
+      return <Loader/>;
+    }
+    const options = this.state.data.fieldOptions;
+    const fields = [
+      {label: 'CandID', show: true, filter: {
+        name: 'CandID',
+        type: 'text',
+      }},
+      {label: 'PSCID', show: true, filter: {
+        name: 'PSCID',
+        type: 'text',
+      }},
+      {label: 'Site', show: true, filter: {
+        name: 'Site',
+        type: 'select',
+        options: options.site,
+      }},
+      {label: 'Visit Label', show: true, filter: {
+        name: 'VisitLabel',
+        type: 'select',
+        options: options.visitLabel,
+      }},
+      {label: 'Project', show: false, filter: {
+        name: 'Project',
+        type: 'multiselect',
+        options: options.project,
+      }},
+      {label: 'Subproject', show: true, filter: {
+        name: 'Subproject',
+        type: 'multiselect',
+        options: options.subproject,
+      }},
+      {label: 'Appointment Type', show: true, filter: {
+        name: 'Appointment Type',
+        type: 'select',
+        options: options.AppointmentType,
+      }},
+      {label: 'Date', show: false, filter: {
+        name: 'Date',
+        type: 'date',
+      }},
+      {label: 'Time', show: false, filter: {
+        name: 'Time',
+        type: 'time',
+      }},
+      {label: 'Earliset Date >=', show: false, filter: {
+        name: 'earlyDate',
+        type: 'date',
+        comparison: 'greaterthanorequal',
+      }},
+      {label: 'Latest Date <=', show: false, filter: {
+        name: 'lateDate',
+        type: 'date',
+        comparison: 'smallerthanorequal',
+      }},
+      {label: 'Starts At', show: true},
+      {label: 'Data Entry Status', show: true},
+      {label: 'Edit', show: true,
+       name: 'edit',
+      },
+      {label: 'Delete', show: true, name: 'delete',
+      },
+      {label: 'AppointmentTypeName', show: false,
+        name: 'AppointmentTypeName',
+      },
+    ];
+    const actions = [
+      {name: 'addSchedule', label: 'Add Schedule', action: this.openModal},
```

Comment:
```suggestion
      {name: 'addSchedule', label: 'Add Appointment', action: this.openModal},
```

I think this should really say 'Add Appointment'. The set of appointments is the schedule